src/test/java/uk/gov/hmcts/reform/profilesync/domain/UserProfileTest.java

```diff
@@ -60,4 +60,11 @@ public void testSetGetValues() {
 
         assertThat(profile.getIdamStatus()).isEqualTo(IdamStatus.ACTIVE.name());
     }
+
+    @Test
+    public void testBuilderToString() {
+        String profile = UserProfile.builder().toString();
+
```

Comment:
make this as well test_ to make uniformity

Reply:
this has been addressed in the following commit: https://github.com/hmcts/rd-profile-sync/pull/276/commits/28a22a3468a439c9b380a79c534d144e0d8b7334